docs/workflow/development.md

```diff
@@ -10,9 +10,63 @@ TODO
 
 TODO
 
-### Board
+### Development Board
 
-TODO
+At HULKs we use a quite heavily modified version of the [KanBan workflow](https://en.wikipedia.org/wiki/Kanban_(development)). The boards' **purpose** is to **visualize** our current development status. The modifications were mainly done to address the fact that we do not have fixed working hours. Limiting the amount of cards that are in progress does not work. Especially for members that are not present on a daily basis.
+
+#### Terms
+
+**Assignee:** The person that is responsible for a card (has the **exclusive** right to move a card(!) as soon as there is one). The assignee's job depends on Kanban column:
+
+- No assignee: *Open*, *Dev Done*, *Done*
+- Assignee works on issue/pull request (if more or less progress happens on it): *In Progress*
+- Assignee is responsible to bring a pull request to `master` (e.g. reviewer, tester): *Discussion*, *Testing*
+
+**Note** that github distinguishes between *reviewers* and *assignees*. We also do that. Kind of. While the *assignee* (in most cases one person) is responsible for the card and bring the branch into the master, the *reviewer(s)* (can be more than one) can review the code at any time in addition to the notes that were given by the *assignee*. In general: The more reviewers the better the resulting code (That being said: feel free to review code).
+
+| *Open* | *In Progress* | *Dev Done* | *Discussion* | *Testing* | *Done* |
+|--------|---------------|------------|--------------|-----------|--------|
+| -- | Developer | -- | Reviewer | Reviewer | -- |
```

Comment:
Two of these columns do not exist anymore, perhaps this whole page needs an overhaul.

Reply:
I would like to delegate this overhaul to you or Luis. Either push on this branch or do it as a separate PR?